docs/volunteering/glossary.md

```diff
@@ -0,0 +1,70 @@
+# Glossary
+
+While we try to use terminology which are clear and/or well known, here's a list
+you can check if you're not sure!
+
+<!---
+Note to editors; please:
+- keep this list in alphabetical order
+- ensure that, within the list, usages of other terms defined in the list are
+  emphasised suitably
+--->
+
+_Competition programme_:
+
+: the overall programme of events and activities which the teams participate in
+
+_Competition year_:
+
+: the 6 months (or so) between _Kickstart_ and the _Competition_ during which
+teams build their robots; see also _SRnnnn_
+
+_Core Team_:
+
+: the group of 8 or so volunteers responsible for delivering a particular year's
+_Competition programme_.
+
+[_GitHub_](https://github.com/srobo):
+
+: a platform for hosting code repositories which we also use to coordinate tasks
```

Comment:
I personally think it would be better to focus on GitHub as a project management tool rather than as a code hosting platform. Code can be scary for new volunteers, and ultimately we do use it primarily for organisation.

Reply:
This definition is accurate for what GitHub is, and what we use it for. 

> GitHub is a development platform

src: https://github.com/

Perhaps something like:

```suggestion
: a development platform, used to host code repositories and to coordinate tasks
```

Reply:
5f141692c693428140e4fdb47fa11a6df0c26db7